tests/debug/test_xdd_debug_init.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Debug output test for XDD.
+#
+# Validate XDD header has worker thread NUMA cpus printed if requested
+#
+# Description - Just writes out to /dev/null using XDD with -debug INIT and verifies
+#               that the NUMA cpus are listed if requested through the debug flag
+#
+# Source the test configuration environment
+#
+source ../test_config
+source ../common.sh
+
+initialize_test
+
+echo "Foobar and I am here motherfucker"
+
+xdd_cmd="${XDDTEST_XDD_EXE} -op write -reqsize 128 -numreqs 1 -targets 1 /dev/null -verbose -debug INIT"
+${xdd_cmd} 2>&1 | grep "bound to NUMA node"
```

Comment:
putting the command into a string only to run it in the next line seems redundant

Reply:
Updated.